### ISSUE TITLE
Eliminate use of word "must" in WCAG notes in 1.4.2, 2.1.2, 2.2.2, and 2.3.1

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#680]
Related to [WCAG Issue 4432: Concern for use of "must" in notes](https://github.com/w3c/wcag/issues/4432). The EN cannot use normative language in non-normative aspects of the standard, such as notes. To make sure the same language is used in both WCAG2ICT and the EN, the WCAG2ICT TF discussed on 5 June 2025 and we think that replacing "must" in the note with "would need to" would be a sufficiently non-normative way to say it.

Thus the note would read:

> Since any part of non-web software that does not meet this success criterion can interfere with a user's ability to use the whole software, all content in the software (whether or not it is used to meet other success criteria) would need to meet this success criterion, including the user interface of the software.

**Comment by maryjom** [comment]
TF agreed on changes to PR #682 in the 13 July meeting which have been incorporated in the code. Just waiting on approval of the PR to merge.

**Comment by maryjom** [comment]
PR #682 has been merged into the editor's draft.
